src/components/Outline.js

```diff
@@ -70,18 +75,21 @@ class Outline extends Component {
   }
 
   render() {
+    const { isHidden } = this.props;
     if (!isEnabled("outline")) {
       return null;
     }
 
     return dom.div(
-      { className: "outline" },
+      { className: classnames("outline", { hidden: isHidden }) },
       dom.ul({ className: "outline-list" }, this.renderFunctions())
     );
   }
 }
 
 Outline.propTypes = {
+  isHidden: PropTypes.bool,
+  sourceText: PropTypes.object,
```

Comment:
:+1: nice! it looks like this might also be a required prop, since the `constructor` and `componentWillReceiveProps` both rely on it. maybe  we should add `isRequired` here?